### PR TITLE
Fix #193

### LINKED ISSUE #193: Missing if condition setting in result.csv
I cannot find any indication of whether the if condition setting is enabled in `result.csv`.

### PR DESCRIPTION
**Pull request by saledouble**
Fix #193.

### CODE REVIEW

**Comment by khatchad** [comment]
Can you upload all information?

On Apr 17, 2019 4:50 PM, Grace Tang <notifications@github.com> wrote:

Related to #193<https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/issues/193>

results.csv (only containing the relevant data):

subject log level not transformed due to if condition   consider if condition
deviation-upload        2       TRUE
deviation-upload        2       TRUE
deviation-upload        2       TRUE
deviation-upload        2       TRUE
deviation-upload        2       TRUE
deviation-upload        2       TRUE
subject log level not transformed due to if condition   consider if condition
deviation-upload        0       FALSE
deviation-upload        0       FALSE
deviation-upload        0       FALSE
deviation-upload        0       FALSE
deviation-upload        0       FALSE
deviation-upload        0       FALSE
________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/pull/194

Commit Summary

  *   Fix #193

File Changes

  *   M edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java<https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/pull/194/files#diff-0> (22)

Patch Links:

  *   https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/pull/194.patch
  *   https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/pull/194.diff

**Comment by saledouble** [comment]
>Can you upload all information?

It is on slack. https://cunyponder.slack.com/archives/D94JKN0G4/p1555534291069900

**Comment by saledouble** [comment]
The entire `results.csv`:


sequence | subject | repo URL | input logging statements | passing logging statements | failures | transformed logging statements | log level not lowered in a catch block | log level not transformed due to if condition | use log category (SEVERE/WARNING/CONFIG) | use log category (CONFIG) | not lower log levels of logs inside of catch   blocks | consider if condition | time (s)
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
1555528165517 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 79 | 0 | 2 | FALSE | FALSE | FALSE | TRUE | 26.022
1555528192266 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 75 | 4 | 2 | FALSE | FALSE | TRUE | TRUE | 15.384
1555528208464 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 78 | 0 | 2 | FALSE | TRUE | FALSE | TRUE | 14.039
1555528222895 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 74 | 4 | 2 | FALSE | TRUE | TRUE | TRUE | 14.175
1555528237431 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 28 | 0 | 2 | TRUE | FALSE | FALSE | TRUE | 13.441
1555528250965 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 27 | 4 | 2 | TRUE | FALSE | TRUE | TRUE | 14.547
sequence | subject | repo URL | input logging statements | passing logging statements | failures | transformed logging statements | log level not lowered in a catch block | log level not transformed due to if condition | use log category (SEVERE/WARNING/CONFIG) | use log category (CONFIG) | not lower log levels of logs inside of catch blocks | consider if condition | time (s)
1555528311142 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 79 | 0 | 0 | FALSE | FALSE | FALSE | FALSE | 16.689
1555528328175 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 75 | 4 | 0 | FALSE | FALSE | TRUE | FALSE | 13.602
1555528342310 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 78 | 0 | 0 | FALSE | TRUE | FALSE | FALSE | 12.42
1555528354980 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 74 | 4 | 0 | FALSE | TRUE | TRUE | FALSE | 12.168
1555528367398 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 28 | 0 | 0 | TRUE | FALSE | FALSE | FALSE | 14.237
1555528381663 | deviation-upload | https://github.com/ponder-lab/deviation-upload.git | 91 | 91 | 0 | 27 | 4 | 0 | TRUE | FALSE | TRUE | FALSE | 13.061

File: edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java [hunk 1]
```diff
@@ -280,7 +279,8 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 									logRejuvenatingProcessor.getLogInvsNotLoweredInCatch().size(),
 									logRejuvenatingProcessor.getLogInvsNotTransformedInIf().size(),
 									this.isUseLogCategory(), this.isUseLogCategoryWithConfig(),
-									this.isNotLowerLogLevel(), resultsTimeCollector.getCollectedTime());
+									this.isNotLowerLogLevel(), this.getValueOfCheckIfCondition(),
```
Comment (khatchad) [inline edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java]: This is incorrect. The `edu.cuny.hunter.log.evaluation.handlers.EvaluationHandler.checkIfCondtion` field should be queried, just like it is for `this.isNotLowerLogLevel()`.

Comment (khatchad) [reply edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java]: BTW, where is this field read?

Comment (saledouble) [reply edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java]: https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/blob/ac4629689008336621213d4bf3ee045388437e7c/edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java#L381

Comment (saledouble) [reply edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java]: Oh, there is a bug here.

Comment (saledouble) [reply edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java]: The variable name is corrected.

**Pull request merged**